docs/reference/cat/shards.asciidoc

```diff
@@ -34,6 +34,10 @@ For <<data-streams,data streams>>, the API returns information about the stream'
 `manage` <<privileges-list-cluster,cluster privilege>> to use this API.
 You must also have the `monitor` or `manage` <<privileges-list-indices,index privilege>>
 for any data stream, index, or alias you retrieve.
+Additionally, to retrieve shard information of restricted indices, you must have `allow_restricted_indices` set to `true`.
+Without the necessary <<roles-indices-priv,index privileges>> for restricted indices,
```

Comment:
```suggestion
Without the necessary <<roles-indices-priv,index privileges>>,
```
On a second thought, this behaviour is general and not specific to restricted indices.